tools/quickstart/quickstart/cli.py

```diff
@@ -239,7 +239,6 @@ def run(
         )
     )
     validator_account.setup_interactively(base_dir=base_dir, chain_dir=chain_dir)
-    validator_account.setup_author_address(setup_name=setup_name, base_dir=base_dir)
```

Comment:
Just comment it out or use a flag with a hint why this does not work and a link to the bug report.